RELEASES.md

```diff
@@ -0,0 +1,90 @@
+## Proposed Release Workflow
+
+Inspired by ["A successful Git branching model"](https://nvie.com/posts/a-successful-git-branching-model/). This article is 10 years old, and now recommends:
+
+> If your team is doing continuous delivery of software, I would suggest to adopt a much simpler workflow (like [GitHub flow](https://guides.github.com/introduction/flow/))
+
+However, we do not have enough automated testing in place to do continuous delivery (CD). Deploying a broken release can have a big impact on partners who are in remote locations, therefore we need a manual QA process for the foreseeable future, in particular for testing offline sync which is [hard to test](https://github.com/digidem/mapeo-mobile/issues/326). We do need the benefits of a more complex workflow with clear release cycles.
+
+### Rules / principles
+
+(avoiding having a `master` branch because of naming issues)
```

Comment:
Shoudl we make 'develop' the 'master' branch then? In GitHub repo settings you can set `develop` as the new master

Reply:
Yes, Github allows you to choose which branch is the default or "base" branch - this is the branch that PRs are automatically made against, unless you change it in the PR (which is what we would do for release PRs, against deploy).